app/src/main/java/org/kontalk/util/Preferences.java

```diff
@@ -299,6 +299,10 @@ public static String getBalloonTheme(Context context) {
         return sBalloonTheme;
     }
 
+    public static boolean getEmojiConverter(Context context){
+        return getBoolean(context, "pref_emoji_converter", false);
```

Comment:
Please add the default value to `res/values/config.xml` (see other preferences) and use that instead of the hard-coded `false`.